priority_queue/priority_queue.md

```diff
@@ -0,0 +1,36 @@
+# priority_queue
+
+**Description :** Priority queues are a type of container, specifically designed such that its first element is always the greatest of the elements.After deleting or inserting into the queue the queue rearranges itself to have the greatest element at the start.There is no ordering among the other elements only the first element is the largest ( hence different from reverse sorting as this is faster in case of upadating and finding largest).
+
+**Example** :
+
+``` cpp
+
+#include <iostream>
+#include <queue>
+#include <vector>
+#include <functional>
+
+int main ()
+{
+  int myints[]= {10,60,50,20};
```

Comment:
Use `std::array<int> myints = {10, 60, 50, 20};`

Reply:
`std:array<int,4> myints;` using this template